docs/src/pages/components/accordion/accordion.md

```diff
@@ -8,7 +8,7 @@ waiAria: https://www.w3.org/TR/wai-aria-practices/#accordion
 
 # Accordion
 
-<p class="description">Accordions contain creation flows and allow lightweight editing of an element.</p>
+<p class="description">Accordions hide and reveal secondary in-page content inside an interactive container.</p>
```

Comment:
The current wording is from https://material.io/archive/guidelines/components/expansion-panels.html. I agree that it's confusing. 

A possible alternative wordin:

```suggestion
<p class="description">The accordion component lets users show and hide vertically sections of related content on a page.</p>
```

Reply:
Riffing off @oliviertassinari's version:

```suggestion
<p class="description">The accordion component allows the user to show and hide sections of related content on a page.</p>
```